### PR TITLE
Catalinaz/tema7

### PR DESCRIPTION
**Pull request by zoldicatalina**

### CODE REVIEW

**Review by ioanaandroflorea — APPROVED** [review]

File: src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    @Test
+    public void homeButton() {
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        WebElement buttonMenu = driver.findElement(By.cssSelector("#menu-toggle"));
+        buttonMenu.click();
+        driver.findElement(By.cssSelector("#sidebar-wrapper > ul > li:nth-child(3) > a")).click();
+        assertEquals("https://katalon-demo-cura.herokuapp.com/", driver.getCurrentUrl());
+    }
+}
+
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java:115]: One blank line it's enough here. Please remove the extra ones. Thanks!

File: src/test/java/teofilursan/tests/seleniumpart1/SeleniumExampleTest.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
 package teofilursan.tests.seleniumpart1;
 
+import ioanandroflorea.driver.BrowserManager;
```
Comment (95teo) [inline src/test/java/teofilursan/tests/seleniumpart1/SeleniumExampleTest.java:3]: Please leave `import teofilursan.driver.BrowserManager;` and not add ioana's one as it's my package. Thanks!

File: src/test/resources/tema7catalinaz/suite.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (95teo) [inline src/test/resources/tema7catalinaz/suite.xml:1]: Use CURAHealthcare from tests/Tema7 package and not the one from teorie/TestNg package. Thanks!

File: src/test/resources/tema7catalinaz/suite.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+<suite name="SuitaTestNG" verbose="1" >
+
+    <test name="CuraHealthTests">
+        <classes>
+            <class name="catalinazoldi.teorie.TestNg.CuraHealthcareTests"/>
```
Comment (95teo) [inline src/test/resources/tema7catalinaz/suite.xml:7]: Use CURAHealthcare from tests/Tema7 package and not the one from teorie/TestNg package. Thanks!

File: src/test/resources/zoldicatalina/groups-tests.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (95teo) [inline src/test/resources/zoldicatalina/groups-tests.xml:1]: You either try to use your groups/classes from teorie/TestNg package with proper changes or you remove this file. Thanks!

File: src/test/resources/zoldicatalina/suite.testing.xml [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (95teo) [inline src/test/resources/zoldicatalina/suite.testing.xml:1]: You either try to use your groups/classes from teorie/TestNg package or you remove this file. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/main/java/test/sample/ParameterSample.java [hunk 7]
```diff
@@ -0,0 +1,4 @@
+package test.sample;
```
[the remark this answers is not in the document]
Comment (zoldicatalina) [reply src/main/java/test/sample/ParameterSample.java:1]: Theo , nu gasesc clasa asta pentru a putea sa o sterg.

File: src/main/java/test/sample/ParameterTest.java [hunk 8]
```diff
@@ -0,0 +1,4 @@
+package test.sample;
+
```
[the remark this answers is not in the document]
Comment (zoldicatalina) [reply src/main/java/test/sample/ParameterTest.java:2]: Nici pe asta.

File: src/test/resources/tema7catalinaz/groups.xml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```
Comment (95teo) [inline src/test/resources/tema7catalinaz/groups.xml:1]: Please rename folder `tema7catalinaz ` to `catalinazoldi`. Thanks!
image

File: src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+    @Test(dataProvider = "appointmentData")
+    public void makeAppointment(String data, String comment, boolean pass) {
+        driver.get("https://katalon-demo-cura.herokuapp.com/");
+        WebElement appointment = driver.findElement(By.id("btn-make-appointment"));
+        appointment.click();
+        completeDetailsForAppointment();
+        Select facility = new Select(driver.findElement(By.id("combo_facility")));
+        facility.selectByVisibleText("Tokyo CURAHealthcare Center");
```
Comment (95teo) [inline src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java:74]: Be careful about the option you choose. You're missing a space -> `Tokyo CURA Healthcare Center`

File: src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+        loginButton.click();
+        System.out.println("Am facut click pe butonul Login din Menu!");
+        driver.findElement(By.id("txt-username")).sendKeys(username);
+        driver.findElement(By.id("txt-password")).sendKeys(password);
+        WebElement buttonLogin = driver.findElement(By.id("btn-login"));
+        buttonLogin.click();
+        if (type.equals("valid")) {
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#appointment > div > div > div > h2")));
+            driver.getCurrentUrl();
```
[the remark this answers is not in the document]
Comment (95teo) [reply src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java:43]: Don't forget to add the assertion. Thanks!

File: src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+        } else
+            wait.until(ExpectedConditions.presenceOfElementLocated(
+                    By.cssSelector("#login > div > div > div.col-sm-12.text-center > p.lead.text-danger")));
```
[the remark this answers is not in the document]
Comment (95teo) [reply src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java:46]: Don't forget to add the assertion. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java (hunk 12 above)
[the remark this answers is not in the document]
Comment (95teo) [reply src/test/java/catalinazoldi/tests/Tema7/CURAHealthcare.java:46]: You're missing the brackets for else branch. It should be like this for else ->
` else {`
 `    wait.until(ExpectedConditions.presenceOfElementLocated(`
 `                   By.cssSelector("#login > div > div > div.col-sm-12.text-center > p.lead.text-danger")));`
 `    assertEquals("https://katalon-demo-cura.herokuapp.com/profile.php#login", driver.getCurrentUrl());`
`}`

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Pull request merged**